apps/explorer/src/components/OwnedCoins/ImageIcon.tsx

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { cva, type VariantProps } from 'class-variance-authority';
```

Comment:
I think we have something similar alrdy here https://github.com/MystenLabs/sui/blob/main/apps/explorer/src/ui/ImageIcon.tsx